Source/WebKit/Shared/WebTouchEvent.h

```diff
@@ -130,6 +130,7 @@ class WebTouchEvent : public WebEvent {
     const Vector<WebPlatformTouchPoint>& touchPoints() const { return m_touchPoints; }
 
     WebCore::IntPoint position() const { return m_position; }
+    void setPosition(WebCore::IntPoint position) { m_position = position; }
```

Comment:
setTransformedPosition?

Reply:
I named this `setPosition()` to match the function in `WebMouseEvent`. I can change both of these but maybe I shouldn't in this PR?